src/App.tsx

```diff
@@ -1,61 +1,116 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+import { TodoList } from './components/TodoList';
+
+const getUserById = (id: number) => {
+  return usersFromServer.find(user => user.id === id) || null;
+};
+
+const todosWithUser: Todo[] = todosFromServer.map(todo => ({
```

Comment:
Consider using `const` instead of `let` for `todosWithUser` since it is not reassigned. 🤔